### ISSUE TITLE
Bombsquad not starting but instead shows error code

### ISSUE THREAD

**Issue by endxrBOI** [efroemling/ballistica#202]
When i start up bombsquad there is a blank white screen and shows this error code 

``` FATAL ERROR: Unhandled exception in BallisticaMain(): Precondition failed: glActiveTexture != nullptr ```

Any help?

**Comment by brostosjoined** [comment]
Yep it's a driver issue consider updating your driver if it still doesn't work downgrade to the first driver software of your GPU

**Comment by Loup-Garou911XD** [comment]
Closing as outdated
